app/javascript/components/editor/TestsGroup.tsx

```diff
@@ -46,30 +32,3 @@ TestsGroup.Header = ({
     <div className="--summary-inner">{children}</div>
   </summary>
 )
-
-TestsGroup.Tests = (): JSX.Element => {
-  const { tests, language } = useContext(TestsGroupContext)
-
-  return (
-    <>
-      {tests.map((test) => (
-        <TestSummary
-          key={test.name}
-          test={test}
-          defaultOpen={test.defaultOpen}
-          language={language}
-        />
-      ))}
-    </>
-  )
-}
-
-TestsGroup.Title = ({ status }: { status: string }): JSX.Element => {
-  const { tests } = useContext(TestsGroupContext)
-
-  return (
-    <>
-      {tests.length} {pluralize('test', tests.length)} {status}
-    </>
-  )
-}
```

Comment:
This was all removed as we now have two different types of test groups: one based on tasks and one based on test status.